### PR TITLE
fix #59 removed video description nullability

### LINKED ISSUE #59: Video Description can't be null in future
### Problem

In future, the description of the videos can no longer be null

### Solution

The mapper must be adapted so that it does not accept videos without a description and the model can then be adapted accordingly so that it no longer accepts null values and the null check can then be removed in the code

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct and Contributing Guidelines

### PR DESCRIPTION
**Pull request by Basler182**
# *removed video description nullability*

## :recycle: Current situation & Problem
#59 

## :gear: Release Notes 
The mapper is adapted so that it does not accept videos without a description and the model is adapted accordingly so that it no longer accepts null values and the null check is removed in the code

## :pencil: Code of Conduct & Contributing Guidelines 

By submitting creating this pull request, you agree to follow our [Code of Conduct](https://github.com/StanfordSpezi/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/StanfordSpezi/.github/blob/main/CONTRIBUTING.md):
- [x] I agree to follow the [Code of Conduct](https://github.com/StanfordSpezi/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/StanfordSpezi/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Basler182** [comment]
as far as i have seen, all videos already have a description. as soon as this is completely fixed, you are welcome to merge it @pauljohanneskraft

**Review by eldcn — APPROVED** [review]
Changed look good from my side 🚀 Let's wait for @pauljohanneskraft confirmation before merging

**Comment by eldcn** [comment]
We just confirmed this with @pauljohanneskraft that field is not nullable anymore, and the description will always be available. Merging :)

**Pull request merged**
